AzureCalling/Views/ParticipantLabelView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@IBDesignable
+class ParticipantLabelView: UIStackView {
+    var insets = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 0)
+
+    @IBInspectable var cornerRadius: CGFloat = 0 {
+        didSet {
+            layer.cornerRadius = cornerRadius
+            layer.masksToBounds = cornerRadius > 0
```

Comment:
What's the purpose of `layer.masksToBounds = cornerRadius > 0`?

Reply:
This was copied over from the previous ParticipantLabel. The purpose of `masksToBounds` is to clip the layer to its boundaries, though it does not make any difference in our case. Removed it to simplify stuff